### PR TITLE
Add TestLocalAddrAndRemoteAddr

### PR DESCRIPTION
**Pull request by bjdgyc**
#### Description
Add some code to test
#### Reference issue
Fixes #...

### CODE REVIEW

File: internal/net/connctx/connctx_test.go [hunk 1]
```diff
@@ -195,3 +195,33 @@ func TestWriteClosed(t *testing.T) {
...
+func (*connAddrMock) SetDeadline(t time.Time) error      { panic("unimplemented") }
+func (*connAddrMock) SetReadDeadline(t time.Time) error  { panic("unimplemented") }
+func (*connAddrMock) SetWriteDeadline(t time.Time) error { panic("unimplemented") }
+
+func TestLocalAddrAndRemoteAddr(t *testing.T) {
+	c := New(&connAddrMock{})
+	al := c.LocalAddr()
+	ar := c.RemoteAddr()
+
+	if al.String() == ar.String() {
```
Comment (at-wat) [inline internal/net/connctx/connctx_test.go:224]: Do you mind changing it to explicitly check each value?
`al.String() == "local_addr"` and `ar.String() == "remote_addr"`

Comment (bjdgyc) [reply internal/net/connctx/connctx_test.go:224]: Ok, I will change it.
By the way, How can i commit it in one pull request.
Because I have pr twice.

Comment (at-wat) [reply internal/net/connctx/connctx_test.go]: OK, I'm going to squash commits and merge.
I'll update the commit message and contributor list in README to fix the lint error.

**Comment by at-wat** [comment]
merged as https://github.com/pion/dtls/commit/513a0e3eef80175e02831380c136f75cb1e84d49

**Pull request closed without merging**